packages/darklang/languageTools/writtenTypesToProgramTypes.dark

```diff
@@ -501,8 +501,10 @@ module Darklang =
               Identifiers.QualifiedConstOrFn.toPT onMissing pm owner currentModule id
 
             match constantOrFn with
-            | Const c -> ProgramTypes.Expr.EConstant(gid (), c)
-            | Fn f -> ProgramTypes.Expr.EFnName(gid (), f)
+            | Ok(Const c) -> ProgramTypes.Expr.EConstant(gid (), c)
+            | Ok(Fn f) -> ProgramTypes.Expr.EFnName(gid (), f)
+            | Error _ ->
+              ProgramTypes.Expr.EVariable (gid ()) constantOrFn.constantOrFn.name
```

Comment:
Added this because we shouldn't allow it to fail during the first pass of parsing. Do you think there is a better way to do it?

Reply:
The alternative, I think, is having this bit return a Result instead, and handling that all the way up